src/components/embedded/EmbeddedMessage.tsx

```diff
@@ -43,13 +43,18 @@ function EmbeddedMessage({ messageId }: { messageId: string }) {
 
 	if (message) {
 		return (
-			<div className='embedded-wrapper'>
-				<MessageCardBase
-					isEmbedded
-					message={message}
-					setViewType={setViewType}
-					viewType={viewType}
-				/>
+			<div>
+				{message.body?.map((item, intemIndex) => (
```

Comment:
There should be a component which renders message body instead of manually iterating over its body